camshare/camshare/AppDelegate.swift

```diff
@@ -11,6 +11,8 @@ UIImage(named: "Image4"), UIImage(named: "Image5"), UIImage(named: "Image6"),
 UIImage(named: "Image7"), UIImage(named: "Image8"), UIImage(named: "Image9"),
 UIImage(named: "Image10"), UIImage(named: "Image11"), UIImage(named: "Image12")]
 
+var imageToBeEdited = UIImage()
```

Comment:
Why is this in AppDelegate? If it has to be that is fine but is there no way we could have it somewhere else ?

Reply:
Actually not using that variable, will remove it.